kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/build/revapi-config.json

```diff
@@ -28,7 +28,26 @@
     "ignores": {
         "revapi": {
             "_comment": "Changes between 7.52.0.Final and the current branch. These changes are desired and thus ignored.",
-            "ignore": []
+            "ignore": [ {
+               "code": "java.method.numberOfParametersChanged",
+                "old": "method javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource::queryProcessesByVariables(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Integer, java.lang.Integer)",
+                "new": "method javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource::queryProcessesByVariables(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Integer, java.lang.Integer, java.lang.String, boolean)",
+                "package": "org.kie.server.remote.rest.jbpm",
+                "classSimpleName": "RuntimeDataResource",
+                "methodName": "queryProcessesByVariables",
+                "elementKind": "method",
+                "justification": "https://issues.redhat.com/browse/JBPM-9714"
+              },
+              {
+                "code": "java.method.numberOfParametersChanged",
+                "old": "method javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource::queryUserTasksByVariables(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Integer, java.lang.Integer)",
+                "new": "method javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource::queryUserTasksByVariables(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Integer, java.lang.Integer, java.lang.String, boolean)",
+                "package": "org.kie.server.remote.rest.jbpm",
+                "classSimpleName": "RuntimeDataResource",
+                "methodName": "queryUserTasksByVariables",
+                "elementKind": "method",
+                "justification": "https://issues.redhat.com/browse/JBPM-9714"
+              }]
```

Comment:
this should not be needed anymore

Reply:
@elguardian Note that the class is RuntimeDataResource, that can be modified without breaking backward compatibility because is the annotated REST resource class, and I hope no one is directly invoking it (doing it will be a mistake). I guess the revapi is there to make sure that nobody changes the resource accidentally. The two parameters added are optional query params.